### PR TITLE
readd text bubbles styles

### PR DESCRIPTION
**Pull request by billythedummy**

### CODE REVIEW

File: dist/output.css [hunk 1]
```diff
@@ -585,6 +585,31 @@ select {
   outline: 1px auto -webkit-focus-ring-color;
 }
 
+.message_bubble {
+  width: -moz-fit-content;
+  width: fit-content;
+  max-width: 80%;
```
Comment (fplord) [inline dist/output.css:591]: can you add max-height too, for bubbles that have very long and thin image messages?

Comment (billythedummy) [reply dist/output.css:591]: sorry could you provide an example of a long and thin image message? What should the max height be set to?

Comment (fplord) [reply dist/output.css:591]: e.g. screenshot of my phone

Comment (fplord) [reply dist/output.css:591]: let's say max-height should be 50% of the layout viewport?

Comment (fplord) [reply dist/output.css:591]: dont worry about it if too bothersome

**Pull request merged**